BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -180,4 +170,13 @@
  **/ 
 - (void)setSharedLinkStorageDelegate:(id <BOXSharedLinkStorageProtocol>)delegate;
 
+/**
+ *  API base URLs.
+ **/
++ (NSString *)APIBaseURL;
```

Comment:
Yeah, I saw that it was already that way. Not a priority right now and for this PR, I don't think. But just calling it out so it's in our consciousness. Ultimately, and as we introduce new things, we want to be unambiguous.